Dockerfile

```diff
@@ -1,16 +1,16 @@
-FROM golang:1.12-alpine
+FROM golang:1.12.9-alpine
```

Comment:
This is a degradation from the previous version of the line because "golang:1.12" resolves to the latest 1.12 version. E.g. if the latest version is 1.12.10 it will get that while 1.12.9 will not.

Furthermore lets use "golang:1-alpine" according to our latest guidelines.